api/src/main/java/org/openmrs/module/fhir2/FhirTask.java

```diff
@@ -56,35 +59,72 @@
 	@Column(name = "task_id")
 	private Integer id;
 	
+	/**
+	 * The current status of the task.
+	 */
 	@Column(name = "status")
 	@Enumerated(EnumType.STRING)
 	private TaskStatus status;
 	
+	/**
+	 * An explanation as to why this task is held, failed, was refused, etc.
+	 */
 	@Column(name = "status_reason")
 	private String statusReason;
```

Comment:
It might be worth dropping this field. It's supposed to be a `CodeableConcept`, but there's no real terminology binding, so it's hard to assess what we need to store.